tests/fields/test_decimal128_field.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from bson.decimal128 import Decimal128
+
+from mongoengine import *
+from tests.utils import MongoDBTestCase
+
+
+class TestDecimal128Field(MongoDBTestCase):
```

Comment:
please add a test that validates what is being written to the database
like we have in EnumField

```
    def test_storage(self):
        model = ModelWithEnum(status=Status.NEW).save()
        assert get_as_pymongo(model) == {"_id": model.id, "status": "new"}
```

Reply:
739cccf212b14ffa09c46f1e9def1bc883b2c8b4